lib/util/resolve.js

```diff
@@ -10,7 +10,21 @@ if(!legacy) { // account for bug in Node v8.9.4 and below
 	}
 }
 
-module.exports = function resolveModulePath(filepath, rootDir) {
+module.exports = function resolvePath(filepath, referenceDir, { enforceRelative } = {}) {
```

Comment:
This constitutes a breaking change, but I'm pretty sure nobody else is using `resolveModulePath`.

Reply:
Yep, that's fine.